### PR TITLE
War! What is it good for?

### PR DESCRIPTION
**Pull request by warior4356**
Who does not want to be Rambo? With warior4356's patent pending war in a box you can! Dispensing hot lead in quantities voluptuous enough to make a death commando blush, you are perfectly gear to get that shuttle called right away and take it all yourself!

The box is hijack only, bartender only, and costs 20TC.

The idea behind it is the bartender will just step into the back room and then walkout loaded to kill everyone in his path.

🆑 Warior4356:
add: Box O' Guns.
/🆑
guns

Update: Items have been toned down to a more reasonable level, the implanter is a adrenals implant.

### CODE REVIEW

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -249,6 +249,15 @@ var/list/uplink_items = list()
 	cost = 6
 	job = list("Bartender")
 
+/datum/uplink_item/jobspecific/assult_box
+	name = "Box O' Guns"
```
Comment (KasparoVy) [inline code/datums/uplink_item.dm]: assault

File: code/datums/uplink_item.dm [hunk 2]
```diff
@@ -249,6 +249,15 @@ var/list/uplink_items = list()
 	cost = 6
 	job = list("Bartender")
 
+/datum/uplink_item/jobspecific/assult_box
```
Comment (KasparoVy) [inline code/datums/uplink_item.dm]: assault

Comment (warior4356) [inline code/datums/uplink_item.dm]: Woops! Fixing!

**Comment by Markolie** [comment]
What can possibly go wrong?

**Comment by Danielhuren** [comment]
for the memes

**Comment by Alex6511** [comment]
👍

**Comment by Jaywings** [comment]
Hah, I like this.

**Comment by Necaladun** [comment]
wayyy too much stuff IMO for 20tcs. Nanites are also...wayy too good.

I'd cut it to 1 main gun and 1 sidearm. The emag seems to go against the whole murdebone aspect of it too. If you're committing to shooting the hell outta everyone, emag isn't needed.

**Comment by Danielhuren** [comment]
neca you need to emag the shuttle console not to mention have you ever tried to hijack going that loud there will be like 30 people going after him

**Comment by Danielhuren** [comment]
i cant argue the nanites though

**Comment by Danielhuren** [comment]
but then again one tazer and hes down without it

**Comment by Danielhuren** [comment]
perhaps somthing to substitute the nanites like adrenal

**Comment by MrFroztee** [comment]
> neca you need to emag the shuttle console not to mention have you ever tried to hijack going that loud there will be like 30 people going after him

You don't need to emag the shuttle if everyone is dead

**Comment by MrFroztee** [comment]
Besides, with an emag as extra access, honestly, it'll just become a man hunt led by an Adminizine fueled man with a bulldog, grenade launcher, and other unnecessary overpowered weapons.

**Comment by Jovaniph** [comment]
This idea would be better if it you add the War song on MIDI.

**Comment by AffectedArc07** [comment]
L6 saw costs 40 TC. Give it to the bartender plus loads of other crap for 20tc

**Comment by Fraility1** [comment]
remove the L6 saw, headset and emag((possibly)

swap the deagle for a stechkin (don't forget to swap ammo)
swap nanites for adrenaline implant


i am conflicted about the emag as any good ai seeing you with all this will just bolt you in somewhere. so it would be useless anyway

**Comment by scrubmcnoob** [comment]
I get the rambo idea but holy guns that's a ton for only 20 TC.
Couldn't other traitors just give a bartender, who is a traitor, 20 TC and then the other traitors will have their own arsenal to dispense the whole crew and they can all claim to be aiding the bartender in his hijack objective.
As @Markolie said, what can possibly go wrong with all these guns.

**Comment by Spacemanspark** [comment]
Holy shit what in the flying-

**Comment by Necaladun** [comment]
I also reaaaally don't like the idea of setting a precedent that hijack+20tcs is this much stuff.

**Comment by Crazylemon64** [comment]
this is kind of a bit much for the cost

**Comment by Lady-Luck** [comment]
Nix everything that isn't a gun or explosive. If you want them to have more access put some c4 in there.

**Comment by tylerh95** [comment]
I like it because it's a nice hijack set, however needs to be toned down slightly. Get rid of the e-mag, the headset, and the holoparasite.

**Comment by fludd12** [comment]
Cut down on the supplies, like the others said, but not by an extreme amount. Remove the nanites entirely, though - adminodrazine by any other name smells just as OP. Make it play the "War, what is it good for" soundfile when opened like @Jovaniph said, since loud and proud is the point of this anyway, and I'll be in support.

**Comment by trexter555** [comment]
take a few things out and it's pretty neat

**Comment by Xantholne** [comment]
But can I be Frank Castle? Former Sol Space Marine who watched his Wife, Son, and Daughter die infront of him thanks to Nanotrasen and Syndicate activities and want to get payback at those sick bastards for it?

**Comment by Danielhuren** [comment]
as a person who has tried to hijack going loud i can tell you now that this loadout would not be enough without the bowman headset one flashbang and there down and droping there gun i get the removing the emag after looking at the code but i would like to point out that even if they do have an excess its about the theatrics of the event this is BARTENDER only and they need the hijack to buy it yes toning it down would be a good idea but you guys are viewing this as a normal hijack and its not anyone can hijack by emaging the shuttle doors and then breaking through the window and emaging the console

**Comment by Danielhuren** [comment]
also for those that are pointing out the saw is 40 tc/ should remove it i would like to point out that this is not a nukie he does not have 4 friends with him at round start with equally devistating equipment and as jovan said if they add a sound clip hes gonna be validhunted by everyone on the station

**Comment by Danielhuren** [comment]
also on a side note it would be easy for an admin to test this if they wanted to see how op it is just do a custom event and adminbus the items in next time someone you trust is barkeep

**Comment by Jovaniph** [comment]
Testing it out as a custom event is a great idea. As long as one of the admins has the soundtrack loaded up and a willing bartender to attempt to hijack with lots of guns...I'm down to be the test subject =3.

**Comment by Aracino** [comment]
Bitchpls I could take this station down with an edagger

**Comment by Markolie** [comment]
Please update the changelog to accurately reflect these changes - "Box of War" doesn't really help users viewing the changelog.

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 3]
```diff
@@ -217,3 +217,17 @@
 		B.deity_name = "Success"
 		B.icon_state = "greentext"
 		B.item_state = "greentext"
+
+/obj/item/weapon/storage/box/syndie_kit/assault_box
+	name = "A box. An unassuming box of WAR!"
+
+/obj/item/weapon/storage/box/syndie_kit/assault_box/New()
+	..()
+	new /obj/item/weapon/storage/backpack/duffel/syndie/ammo/assault(src)
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: This is incredibly bad form to cram a duffel bag into a a box.

**Comment by tristan1333** [comment]
Difference between an op and a traitor with an L6 Saw, the Nukie is in a crew, the traitor is alone. Taser > L6  Saw. It takes only 1 suicidal assistant with a stunprod to end his round. It should stay.

File: code/game/objects/items/weapons/storage/uplink_kits.dm (hunk 3 above)
Comment (taukausanake) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: What about a larger box? We have those but I forget what fits inside them. I just use them for more item storage

Comment (Danielhuren) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: well an empty dufflebag could easily fit! not to mention realism in what people put in boxes isent really a thing or you wouldent have alot of items

**Comment by KingPhilipIII** [comment]
A smart Rambo bartender will hold off until robotics gets no-drop implants and he can convince them to give him one.

**Comment by Fox-McCloud** [comment]
Internally rejected.

I'd also point that hijack items are typically supposed to be high risk, high reward and force you to go ham. This package is certainly a "no holds bar" package, but it doesn't really encourage the behavior; His Grace *forces* you into going ham or you'll die. This is just giving the bartender lots of tools for murder; it's not even particularly unique or thematically centered around him.

**Pull request closed without merging**